cmake/toolchain/llvm/generic.cmake

```diff
@@ -20,7 +16,11 @@ if("${ARCH}" STREQUAL "arm")
   set(triple arm-none-eabi)
```

Comment:
We could consider if it would be beneficial to use the:
https://clang.llvm.org/docs/CrossCompilation.html#target-triple
sub for arm, like: `v6m`, `v7m`, `v7a`, ...

although impact might be limited as we already are setting `-mcpu`.